scripts/toolchain/install-toolchain.sh

```diff
@@ -218,6 +244,7 @@ if ${install_all}; then
 
     install_wasm_opt
     install_wasm_pack
+    install_protobuf
```

Comment:
Missing version printout at the end of the script. Also adding the new deployment to the doc in the start of the script.

Reply:
Added.